### PR TITLE
script to publish final files; templatize readmes

### PR DESCRIPTION
**Pull request by markusicu**
New publication script for assembling the data files (as far as they are kept in this repo) for a Unicode release.

The various ReadMe.txt files are now templatized, and the publication scripts fill in some variables. I chose `find` and `sed` because it was easy.

### CODE REVIEW

**Review by Manishearth — DISMISSED** [review]
ooh this seems useful

**Review by nedley — APPROVED** [review]

**Review by eggrobin — APPROVED** [review]

**Review by josh-hadley — DISMISSED** [review]

**Comment by markusicu** [comment]
I reviewed the changes once more myself and found a couple of minor things. I just fixed them and pushed an amended commit. Please re-approve, and sorry for the churn.

**Comment by markusicu** [comment]
I see that we don't auto-generate a message here for comparing across the force-push. Here is what I just changed:
https://github.com/markusicu/unicodetools/compare/9429563b..f1c5612e

**Comment by eggrobin** [comment]
> we don't auto-generate a message here for comparing across the force-push

No, but GitHub does, to the right here:
image

**Review by eggrobin — APPROVED** [review]

File: unicodetools/data/security/dev/ReadMe.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Unicode Security Data (Confusables and Identifiers)
-# © 2023 Unicode®, Inc.
+# © COPY_YEAR Unicode®, Inc.
 # Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
 # For terms of use, see https://www.unicode.org/terms_of_use.html
 
-This directory contains the data files for Version 15.1.0 of
+This directory contains the data files for Version UNI_VER of
```
Comment (nedley) [inline unicodetools/data/security/dev/ReadMe.txt:6]: Do we want `PUB_STATUS` for this file?

**Review by nedley — APPROVED** [review]

File: unicodetools/data/security/dev/ReadMe.txt (hunk 1 above)
Comment (markusicu) [reply unicodetools/data/security/dev/ReadMe.txt:6]: In this PR I just want to templatize what we have.
Asmus is looking at revising the readmes overall, and Anne might want to chime in on copyright, terms of use, etc.
Let's make those kinds of content changes separately.

**Pull request merged**